bossdata/spec.py

```diff
@@ -16,6 +16,36 @@
 import astropy.table
 
 
+def get_fiducial_pixel_index(wavelength):
+        """
+        Convert a wavelength to a fiducial pixel index.
+
+        Args:
+            wavelength(float): Input wavelength in Angstroms.
+
+        Returns:
+            numpy.ndarray: Array of floating-point indices relative to the fiducial
+                wavelength grid.
+        """
+        return (np.log10(wavelength) - _fiducial_log10lam0)/_fiducial_coef
```

Comment:
Grouping bins (downsampling) is a special case of resampling so I think most of the work (weighted combinations of flux, ivar, dispersion) belongs in the speclite package, not here. Since downsampling is based on:
```
ds_index = (index - index0) // nbins
```
the API only needs ``index0`` and ``nbins`` as input, in addition to the original wavelength grid of the input data, in order to do its work.  I think this makes a generalized `get_fiducial_pixel_index` unecessary - do you agree?